samples/dnn/text_detection.py

```diff
@@ -0,0 +1,159 @@
+# Import required modules
+import cv2 as cv
+import math
+import argparse
+import time
+import numpy as np
+
+############ Add argument parser for command line arguments ############
+parser = argparse.ArgumentParser(description='Use this script to run text detection deep learning networks using OpenCV.')
+# Input argument
+parser.add_argument('--input', help='Path to input image or video file. Skip this argument to capture frames from a camera.')
+# Model argument
+parser.add_argument('--model', required=True,
+                    help='Path to a binary .pb file of model contains trained weights.'
+                    )
```

Comment:
What about these comments? I still think that `parser.add_argument('--model'` is enough and there is no need to replicate it in comments.

Please also do not put `)` to the new lines

```python
help='Path to a binary .pb file of model contains trained weights.')
```
looks fine.